monGARS/core/llm_integration.py

```
@@ -1052,53 +1135,129 @@ async def _ensure_local_models(self) -> None:
+                    messages=messages,
+                    options=options,
+                )
+
+            try:
+                response = await self._ollama_cb.call(_ollama_request)
+            except CircuitBreakerOpenError:
+                logger.warning(
+                    "llm.ollama.breaker_open",
+                    extra={"task_type": task_type, "model": model_definition.name},
+                )
+            except Exception as exc:  # pragma: no cover - defensive logging
```

Ruff (0.14.4)

Local variable `exc` is assigned to but never used

Remove assignment to unused variable `exc`

(F841)